backend/src/main/java/de/neuefische/koheis/backend/chractercards/CharacterCardService.java

```diff
@@ -0,0 +1,20 @@
+package de.neuefische.koheis.backend.chractercards;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CharacterCardService {
+    private final CharacterCardRepository characterCardRepository;
+
+
+    public CharacterCardService(CharacterCardRepository characterCardRepository){
```

Comment:
you could use `@RequiredArgsConstructor` from lombok instead of this constructor here

Reply:
👍